common/changes/@rushstack/heft/sachinjoseph-updateTarballIntegrityHashDuringVersionBump_2020-07-02-08-27.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Update temp project tarball integrities during rush bump",
+      "type": "none"
```

Comment:
Since the constructor is a breaking change, you may wanna be more descriptive and make it a minor/at least a patch bump type.